sqlalchemy-netezza/test/test_suite.py

```diff
@@ -37,11 +37,57 @@ def _netezza_temp_table_keyword_args(cfg, eng):
     TableDDLTest as _TableDDLTest,
     IntegerTest as _IntegerTest,
     ComponentReflectionTest as _ComponentReflectionTest,
+    CompositeKeyReflectionTest as _CompositeKeyReflectionTest,
     DateHistoricTest as _DateHistoricTest,
     DateTimeHistoricTest as _DateTimeHistoricTest,
     TimestampMicrosecondsTest as _TimestampMicrosecondsTest,
+    QuotedNameArgumentTest as _QuotedNameArgumentTest,
 )
 
+class QuotedNameArgumentTest(_QuotedNameArgumentTest):
```

Comment:
should try to minimize skipped tests

Reply:
CompositeKeyReflectionTest  and QuotedNameArgumentTest are the newly added testcases in sqlalchemy since nzalchemy was released with pyodbc. Both these testcases are failing with pyodbc and nzpy, I didn't investigate failures. We can have a look and provide fix in separate PR.